docs/en/reference/sql/dql/GROUP_BY_CLAUSE.md

```diff
@@ -17,25 +15,23 @@ SELECT select_expr [,select_expr...] FROM ... GROUP BY ...
 
 ## Boundary Description
```

Comment:
@michelle-qinqin I just notice this`boundary description` that appears many times in different docs. This also seems a bit weird (like a Chinglish ...). Maybe we just change it to `Description`?

Reply:
that's ok with me~
But can I create a new branch and a new PR to change this statement in all related files later?
Since there are many unmerged branches that might have this problem, and there are both Chinese and English docs using `boundary description`, it would be more convenient and safer for me to check all docs for once specifically for this problem.

Reply:
yes, you can fix this issue later in a single PR

Reply:
yes, you can fix this issue later in a single PR